### PR TITLE
chore: Moves Generating Stories

### PR DESCRIPTION
**Pull request by redreceipt**
## TODO:

- [ ] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [ ] No new warnings in tests, in storybook, and in-app
- [ ] Upload GIF(s) of iOS and Android
- [ ] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "postinstall": "lerna run build --parallel",
     "generate-docs": "lerna run generate-docs",
     "generate-stories": "rnstl --searchDir ./src --pattern **/*.stories.js",
-    "storybook": "(cd ApollosStorybook && yarn && yarn ios)"
+    "storybook": "(cd ApollosStorybook && yarn && yarn generate-stories && yarn ios)"
```
Comment (vinnyjth) [inline package.json:32]: This won't work here, because you're `cd`ed into the wrong dir at this point :) It will also need to be `yarn lerna run generate stories`

**Review by vinnyjth — APPROVED** [review]

**Comment by vinnyjth** [comment]
Best approach thus far.
